### PR TITLE
Add a public method to focus the component

### LINKED ISSUE #150: [Feature Request] Add a public method to focus the component
**Is your feature request related to a problem? Please describe.**
I'm trying to find a way to programmatically expand the result dropdown list  

**Describe the solution you'd like**
I would like to see a public method that can be called that will focus the input box and expand the dropdown list

**Describe alternatives you've considered**
I have tried to set focus using JSInterop to see if that would trigger it, with no luck.

**Additional context**
I have a bootstrap modal that contains the Typeahead control along with Save and Cancel buttons. I'm trying to have the Typeahead control expand when the modal opens.

### PR DESCRIPTION
**Pull request by kiasyn**
Fixes #150

### CODE REVIEW

**Review by chrissainty — CHANGES_REQUESTED** [review]
This looks great @kiasyn. Just need a couple of things before I can merge it though:

1. Could you update the Blazor Server sample with the example as well
2. Could you add a some details to the readme so people know this feature exists and how to use it

Once these bit are done, this can be merged straight away

**Comment by zippo-lighter** [comment]
@chrissainty what is blocking the merge into the main repository?

**Comment by chrissainty** [comment]
@zippo-lighter As per the comment directly above yours states, there is no documentation or samples for this work.

**Review by chrissainty — APPROVED** [review]

**Pull request merged**
